architecture/globalplatform_api.rst

```diff
@@ -132,7 +135,8 @@ to operate with the data cache:
     TEE_Result TEE_CacheInvalidate(char *buf, size_t len);
 
 These functions are available to any Trusted Application defined with the flag
-``TA_FLAG_CACHE_MAINTENANCE`` sets on. When not set, each function returns the
+``TA_FLAG_CACHE_MAINTENANCE`` sets on, see :ref:`ta_property_cache_maintenance`.
+When not set, each function returns the
```

Comment:
Can you wrap this and the rest of the lines in this section on 80 character?

Reply:
ok